libs/martech/attributes.js

```diff
@@ -20,13 +20,22 @@ export function decorateDefaultLinkAnalytics(block, config) {
     let header = '';
     let linkCount = 1;
 
-    let headerSelector = 'h1, h2, h3, h4, h5, h6, .tracking-header';
-    const headers = block.querySelectorAll(headerSelector);
-    if (!headers.length) headerSelector = `${headerSelector}, b, strong`;
-    block.querySelectorAll(`${headerSelector}, a:not(.video.link-block), button`).forEach((item) => {
+    const headerSelector = 'h1, h2, h3, h4, h5, h6';
+    let analyticsSelector = `${headerSelector}, .tracking-header`;
+    const headers = block.querySelectorAll(analyticsSelector);
+    if (!headers.length) analyticsSelector = `${analyticsSelector}, b, strong`;
+    block.querySelectorAll(`${analyticsSelector}, a:not(.video.link-block), button`).forEach((item) => {
       if (item.nodeName === 'A' || item.nodeName === 'BUTTON') {
         if (item.classList.contains('tracking-header')) {
           header = processTrackingLabels(item.textContent, config, 20);
+        } else if (!header) {
+          const section = block.closest('.section');
+          if (section.className.includes('-up') || section.classList.contains('milo-card-section')) {
+            const previousHeader = section?.previousElementSibling.querySelector(headerSelector);
```

Comment:
I'd guess the optional chaining should go after `previousElementSibling`, since on L33 you're already reading properties for `section`?

Reply:
@overmyheadandbody agreed. probably safer. added